src/website2/src/views/careers/DetailsPage.tsx

```
@@ -8,108 +7,165 @@ import { FiArrowLeft } from 'react-icons/fi';
+        {careerDetails.descriptions &&
+          careerDetails.descriptions.length > 0 && (
+            <div className="space-y-4">
+              {' '}
+              <h2 className="text-2xl font-semibold text-gray-900">
+                Job Description
+              </h2>{' '}
+              {careerDetails.descriptions.map((desc: any, index: number) => (
+                <div
+                  key={index}
+                  className="prose prose-lg max-w-none text-gray-700 leading-relaxed"
+                  dangerouslySetInnerHTML={{ __html: desc.description || '' }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)